src/Query/Grouper.ts

```diff
@@ -1,12 +1,37 @@
 import type { Task } from '../Task';
 
 /**
- * A naming function, that takes a Task object and returns the corresponding group property name
+ * A group-naming function, that takes a Task object and returns zero or more
+ * corresponding group property names.
```

Comment:
as discussed in Discord, I feel like `property` here is not the correct object to reference. For instance, below `would return ['tag1', 'tag2']` are not properties.

Reply:
Later versions of this code may have explicit types for the concepts, which would make it all clearer...